Extensions/Tokens/Extensions/TokenProviderExtensions.cs

```diff
@@ -13,6 +13,12 @@ public static class TokenProviderExtensions
             LocalizedString description) where TPart : ContentPart =>
             describeFor.Token($"{typeof(TPart).Name}.{tokenName}", name, description);
 
+        public static DescribeFor ContentPartToken<TPart>(
+            this DescribeFor describeFor,
+            string tokenName,
+            LocalizedString nameAndDescription) where TPart : ContentPart =>
+            describeFor.Token($"{typeof(TPart).Name}.{tokenName}", nameAndDescription, nameAndDescription);

```

Comment:
You should rather reuse the existing method above instead of duplicating its code here.

Reply:
Yeah, I should definitely do that!